cirq/circuits/circuit_dag_test.py

```diff
@@ -109,9 +109,14 @@ def test_from_circuit():
     assert sorted(circuit.all_qubits()) == sorted(dag.all_qubits())
 
 
+class TestDevice(cirq.Device):
+    def validate_operation(self, operation: cirq.Operation) -> None:
```

Comment:
Can't you use cirq.UNCONSTRAINED_DEVICE?

Reply:
Ah, nice, let me try that